tutorials/tts/NeMo_TTS_Primer.ipynb

```diff
@@ -144,7 +144,7 @@
     "While this is the most common structure, there may be fewer or additional steps depending on the use case. For example, some languages do not require G2P and can instead rely on the model to convert raw text/graphemes to spectrogram.\n",
     "\n",
     "<div>\n",
-    "<img src=\"https://raw.githubusercontent.com/NVIDIA/NeMo/tts_primer/tutorials/tts/images/tts_pipeline.png\" width=\"500\", height=\"500\"/>\n",
+    "<img src=\"https://raw.githubusercontent.com/NVIDIA/NeMo/main/tutorials/tts/images/tts_pipeline.png\" width=\"500\", height=\"500\"/>\n",
```

Comment:
These should probably be stable, which means they'll be broken for now. The alternative is to leave it as `main`, but you have to update it to stable in the next rc

Reply:
actually, double check with jocelyn or som

Reply:
I'd say keep it as `main` for now; I'd rather have the images linked properly with a small TODO than have it broken (especially as a primer tutorial).